Classification/FeatureSelection/src/test/java/org/tribuo/classification/fs/MIMTest.java

```diff
@@ -144,7 +144,10 @@ public void mimTest() {
         assertEquals(5,scores.size());
 
         assertEquals(Arrays.asList("E","B","C","D","A"),sfs.featureNames());
-        assertEquals(Arrays.asList(1.0, 0.1887218755408671, 0.1887218755408671, 0.0487949406953985, 0.0),sfs.featureScores());
+        List<Double> expectedScores = Arrays.asList(1.0, 0.1887218755408671, 0.1887218755408671, 0.0487949406953985, 0.0);
```

Comment:
```suggestion
        double[] expected = new double[]{1.0, 0.1887218755408671, 0.1887218755408671, 0.0487949406953985, 0.0};
        assertArrayEquals(expected, Util.toPrimitiveDouble(sfs.featureScores()), 1e-12);
```

This is a little shorter and will give a slightly better error message when the test fails.